src/Module.php

```diff
@@ -21,20 +21,18 @@
 
 namespace ZfcRbac;
 
-/**
- * Class ModuleConfig
- *
- * @author  Florent Blaison <florent.blaison@gmail.com>
```

Comment:
could you add him to https://github.com/ZF-Commons/zfc-rbac/blob/master/composer.json#L13